examples/prism-bbj.html

```diff
@@ -0,0 +1,83 @@
+<h2>Comments</h2>
+<pre><code>rem Single line comment
+rem /**
+rem  * This file is part of the X plugin.
+rem  *
+rem  * For the full copyright and license information, please view the LICENSE
+rem  * file that was distributed with this source code.
+rem  */</code></pre>
```

Comment:
This seems weird...

You don't have to do these kinds of one-feature-at-a-time examples btw. You can also just have one example that shows off the language. [Example](https://github.com/PrismJS/prism/blob/master/examples/prism-armasm.html).

Reply:
> Also, is there documentation for BBj? The best I found was [this](https://documentation.basis.cloud/BASISHelp/WebHelp/contents/BBj-Documentation-contents.htm).

That's the official documentation site

Reply:
> This seems weird...
> 
> You don't have to do these kinds of one-feature-at-a-time examples btw. You can also just have one example that shows off the language. [Example](https://github.com/PrismJS/prism/blob/master/examples/prism-armasm.html).

Why ?! I use the [java-example](https://github.com/hyyan/prism/blob/master/examples/prism-java.html) as a template

Reply:
Sorry. The Java example is pretty old. Nowadays, we typically just provide one example because it's easier for both use writing the example and people viewing examples.